pkgs/applications/audio/mpg123/default.nix

```diff
@@ -1,6 +1,6 @@
 { stdenv
 , fetchurl, alsaLib
-, buildPlatform, hostPlatform
+, hostPlatform
```

Comment:
@bjornfor I have considered that, and wobbled back and forth many times. It's a bit funny in that the stdenv is built from the previous stage, and needs *that stages's* platforms (e.g. what's the build platform of stdenv itself), but then `mkDerivation` obviously needs this stage's platforms. I do admit that, infrastructure aside, `stdenv.hostPlatform` would be easier. I guess despite what I said above about the arguments becoming idiomatic, I hope most derivations can work the same way regardless of cross compilation so such finessing wouldn't be *too* common :).

http://github.com/triton/triton puts then in `stdenv`, I learned recently. I should probably see how they do that, though they didn't make (as far as I can tell) anything like the bootstrapping overhaul I did the end of last year.